Makefile

```diff
@@ -63,6 +67,17 @@ proto/generate:
 proto/clean:
 	rm -rf internal/gen/proto
 
+.PHONY: proto/dev
+proto/dev: build proto/clean proto/generate
+	cp -vrf internal/gen/proto/ts/runme $(RUNME_EXT_BASE)/node_modules/@buf/stateful_runme.community_timostamm-protobuf-ts
```

Comment:
Should `RUNME_EXT_BASE` have a default value? I guess typically it will be in the same parent dir, so maybe:
```sh
ifeq ($(RUNME_EXT_BASE),)
RUNME_EXT_BASE := "../vscode-runme"
endif
```

Or whatever makes sense.